sway/config/output.c

```diff
@@ -329,7 +329,7 @@ static void default_output_config(struct output_config *oc,
 			wl_container_of(wlr_output->modes.prev, mode, link);
 		oc->width = mode->width;
 		oc->height = mode->height;
-		oc->refresh_rate = mode->refresh;
+		oc->refresh_rate = mode->refresh / 1000;
```

Comment:
All of these are integers, should probably convert either `mode->refresh` or `1000` to float.

Reply:
Fixed.